index.js

```diff
@@ -13,7 +13,7 @@ mongoose.connect(
     "mongodb+srv://vishal:12345@cluster0.nqdbc.mongodb.net/ManagementToolForHR?retryWrites=true&w=majority",
     { useNewUrlParser: true, useUnifiedTopology: true }
 );
-
+mongoose.set('useFindAndModify', false);
```

Comment:
resson of using this?

Reply:
I was getting Deprecation Warnings while using ( fineOne() ) that's why I have used this.